catalog/core/catalog-core-impl/filter-proxy/pom.xml

```diff
@@ -53,6 +53,12 @@
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
         </dependency>
+        <dependency>
```

Comment:
might consider moving this to dependency management under the catalog pom.xml since it is used in various places.